base/exports.jl

```diff
@@ -166,7 +166,10 @@ export
     ≠,
     !==,
     ≡,
+    ⩶,
     ≢,
+    !⩵,
+    !⩶,
```

Comment:
These are a bit weird.  If there's no Unicode `≠≠≠`, should we be adding an ASCII–Unicode hybrid symbol?  We don't have any other operators like this.

Reply:
Yes, it is a question I had asked [here](https://github.com/JuliaLang/julia/issues/41462#issuecomment-892931970). It was to make the (graphic) equivalents of `==` & `!=` and `===` & `!==`.
@StefanKarpinski, what would be more julian here?

Reply:
Agree that the hybrid ASCII/Unicode symbols feel over the line. Let's keep this more minimal, can always do more later.